config/config.go

```diff
@@ -1312,6 +1364,8 @@ type RemoteWriteConfig struct {
 	// values arbitrarily into the overflow maps of further-down types.
 	HTTPClientConfig HTTPClientConfig `yaml:",inline"`
 
+	RemoteQueueConfig RemoteQueueConfig `yaml:",inline"`
```

Comment:
I'm not sure we should be exposing this level of detail to users, expecting them to e.g. configure big enough buffers might be a bit much.

This also feels far more flaggy to me.